defaults/main.yml

```diff
@@ -20,6 +20,11 @@ datadog_yum_repo: "https://yum.datadoghq.com/stable/6/{{ ansible_userspace_archi
 datadog_yum_gpgkey: "https://yum.datadoghq.com/DATADOG_RPM_KEY.public"
 datadog_yum_gpgkey_new: "https://yum.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public"
 
+# default zypper repo and keys
+datadog_zypper_repo: "https://yum.datadoghq.com/suse/stable/6/{{ ansible_userspace_architecture }}"
+datadog_zypper_gpgkey: "https://yum.datadoghq.com/DATADOG_RPM_KEY.public"
+datadog_zypper_gpgkey_sha256sum: "00d6505c33fd95b56e54e7d91ad9bfb22d2af17e5480db25cba8fee500c80c46"
```

Comment:
This is never used anywhere, we should remove it.

Reply:
Hi @hush-hush,
This is my fault, I should have been using these variables. This is now fixed